torch/csrc/jit/codegen/cuda/scheduler/pointwise_utils.cpp

```diff
@@ -13,7 +13,9 @@ bool DomainMap::areAllInputIdsMappedTo(TensorView* input_tv, TensorView* tv)
   std::unordered_set<IterDomain*> in_concrete_ids;
   for (auto in_id : input_tv->getMaybeRFactorDomain()) {
     auto concrete = ca_map_.getConcreteMappedID(in_id, IdMappingMode::EXACT);
-    if (!concrete->isBroadcast() && !in_id->isReduction()) {
+    if (!concrete->isBroadcast() && !in_id->isReduction() &&
+        // Ignore trivial reductions
+        !(in_id->isBroadcast() && concrete->isTrivialReduction())) {
```

Comment:
Would `!concrete->isTrivialReduction()` be sufficient here? I can not think of any case where a non-broadcasting ID is mapped to a trivial reduction.

Reply:
Yeah, I think that's sufficient. Well remove it.